marlowe/test/Spec/Marlowe/Marlowe.hs

```diff
@@ -274,20 +276,23 @@ mulAnalysisTest :: IO ()
 mulAnalysisTest = do
     let muliply = foldl (\a _ -> MulValue (UseValue $ ValueId "a") a) (Constant 1) [1..100]
         alicePk = PK $ pubKeyHash $ walletPubKey alice
-        aliceAcc = AccountId 0 alicePk
-        contract = If (muliply `ValueGE` Constant 10000) Close (Pay aliceAcc (Party alicePk) ada (Constant (-100)) Close)
+        contract = If (muliply `ValueGE` Constant 10000) Close (Pay alicePk (Party alicePk) ada (Constant (-100)) Close)
     result <- warningsTrace contract
     --print result
     assertBool "Analysis ok" $ isRight result
 
 
 pangramContractSerialization :: IO ()
 pangramContractSerialization = do
+    let json = toStrict (encodeToLazyText pangramContract)
```

Comment:
this looks like it should also be commented. also, `toStrict . encodeToLazyText` would make me think there is `encodeText` somewhere? No type hint so presumably you can inline this into the `T.putStrLn` part. It's only doing `putStrLn` so it could be a string, doesn't `putStrLn $ encode pangramContract` do what these 2 lines do?

Reply:
also, uncomment after updating doesn't sound very appealing, can't this be done another way like using nix or something? I feel like one day some poor soul is going to sink much time into working out why the tests are failing all of a sudden.

Reply:
Dmytro is going to fix all that.